mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -319,6 +327,37 @@ properties:
                   description: |
                     Number of accelerator cards exposed to the instance.
                   required: true
+          - !ruby/object:Api::Type::Array
+            name: 'boostConfigs'
+            description: |
+              A list of the boost configurations that workstations created using this workstation configuration are allowed to use.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'id'
+                  description: |
+                    The id to be used for the boost config.
+                  required: true
+                - !ruby/object:Api::Type::String
+                  name: 'machineType'
+                  description: |
+                    The type of machine that boosted VM instances will use. Defaults to `e2-standard-4`.
```

Comment:
Can you please copy the whole description of this field `machineType` from API doc here?

https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs#BoostConfig